packages/core/src/node/backend-application.ts

```diff
@@ -101,20 +101,21 @@ export interface BackendApplicationContribution {
 @injectable()
 export class BackendApplicationCliContribution implements CliContribution {
 
-    port: number;
-    hostname: string | undefined;
-    ssl: boolean | undefined;
-    cert: string | undefined;
-    certkey: string | undefined;
-    projectPath: string;
+    projectPath: string = this.appProjectPath();
+
+    port?: number;
+    hostname?: string;
+    ssl?: boolean;
+    cert?: string;
+    certkey?: string;
 
     configure(conf: yargs.Argv): void {
         conf.option('port', { alias: 'p', description: 'The port the backend server listens on.', type: 'number', default: DEFAULT_PORT });
         conf.option('hostname', { alias: 'h', description: 'The allowed hostname for connections.', type: 'string', default: DEFAULT_HOST });
         conf.option('ssl', { description: 'Use SSL (HTTPS), cert and certkey must also be set', type: 'boolean', default: DEFAULT_SSL });
         conf.option('cert', { description: 'Path to SSL certificate.', type: 'string' });
         conf.option('certkey', { description: 'Path to SSL certificate key.', type: 'string' });
-        conf.option(APP_PROJECT_PATH, { description: 'Sets the application project directory', default: this.appProjectPath() });
+        conf.option(APP_PROJECT_PATH, { description: 'Sets the application project directory' });
```

Comment:
Why remove this `default`?

Reply:
I had understood that `BackendApplicationCliContribution.configure/setArguments` wasn't called when this component was used, but turns out it might because the CLI is initialized in the generated app. Will revert this part now that I better understand how it works. I got confused.